honeybee_energy/hvac/allair/_base.py

```diff
@@ -84,6 +85,7 @@ def __init__(self, identifier, vintage='ASHRAE_2019', equipment_type=None,
         self.sensible_heat_recovery = sensible_heat_recovery
         self.latent_heat_recovery = latent_heat_recovery
         self.demand_controlled_ventilation = demand_controlled_ventilation
+        self._user_data = None
```

Comment:
Remove this line since it already happens when initializing the parent class.

Reply:
yeah... slots: this is my first time writing using slots: apologies